apps/angular/src/core/dtos/anime-status.dto.ts

```diff
@@ -0,0 +1,6 @@
+/** List of anime status. */
```

Comment:
This is not a list, this is enum. We usually just describe what enum represents. Please check other enums as well.
```suggestion
/** Anime status DTO. */
```